Content.Shared/Radio/Components/HeadsetComponent.cs

```diff
@@ -15,4 +16,10 @@ public sealed partial class HeadsetComponent : Component
 
     [DataField("requiredSlot")]
     public SlotFlags RequiredSlot = SlotFlags.EARS;
+
+    [DataField("sound")]
+    public SoundSpecifier Sound;
+
+    [DataField("soundChannels")]
+    public HashSet<string> SoundChannels = [];
```

Comment:
Use protoids! I think `ProtoId<RadioChannelPrototype>` will work here. It'll automatically ensure they are valid in the ymal which is nice.

I would also recommend renaming this to something like "SelectedSoundChannels" or something. It's kind of a confusing name right now (At least for me 😆)!

Reply:
Content.Shared\Radio\Components\EncryptionKeyHolderComponent.cs
> [ViewVariables]
> public HashSet<string> Channels = new();

In this component used `string`. And I need to store ID of channel, not ID of prototype.